src/disc/stk/Albany_SideSetSTKMeshStruct.cpp

```diff
@@ -102,9 +105,12 @@ SideSetSTKMeshStruct::SideSetSTKMeshStruct (const MeshSpecsStruct& inputMeshSpec
                                                                 ebn, ebNameToIndex, this->interleavedOrdering));
 
   const Teuchos::MpiComm<int>* mpiComm = dynamic_cast<const Teuchos::MpiComm<int>* > (commT.get());
-  bulkData = Teuchos::rcp(new stk::mesh::BulkData(*metaData, *mpiComm->getRawMpiComm(),
-                                                   stk::mesh::BulkData::NO_AUTO_AURA,
-                                                   false, NULL, worksetSize));
+  stk::mesh::MeshBuilder meshBuilder = stk::mesh::MeshBuilder(*mpiComm->getRawMpiComm());
```

Comment:
I forgot why we don't let GenericSTKMeshStruct create the bulk data in this class...

Reply:
Don't ask me.. :D